shadowreader/classes/app.py

```diff
@@ -92,3 +92,13 @@ def _is_valid_operand(self, other):
                 hasattr(other, "identifier") and
                 hasattr(other, "cur_timestamp")
                 )
+
+    def _validate_base_url(self, base_url):
+        # If base_url ends with a "/", remove it
+        if base_url.endswith('/'):
+            base_url = base_url[:-1]
+
+        # If base_url does not specify http or https, prefix it with http://
+        if not (base_url.startswith('http://') or base_url.startswith('https://')):
+            base_url = f'http://{base_url}'
```

Comment:
Why not default to `https`?

Reply:
I was thinking that an endpoint may not have https enabled, so it would be better to default to http